docs/user/bots.rst

```diff
@@ -2559,15 +2559,22 @@ specified with quotes. Following operators may be used to match events:
 If part of a rule matches the given conditions, the actions enclosed in `{` and
 `}` are applied. By default, all events that are matched or not matched by rules
 in the sieve file will be forwarded to the next bot in the pipeline, unless the
-`drop` action is applied.
+`drop` action is applied. 
+
+**ATTENTION** Math only supported on DateTime objects currently!
```

Comment:
Please move the note to the section below where it belongs to (basic math operations)

Reply:
The sentence is missing a verb.